src/Middleware/RequestBodyParserMiddleware.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace React\Http\Middleware;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class RequestBodyParserMiddleware
+{
+    public function __invoke(ServerRequestInterface $request, $next)
+    {
+        $type = strtolower($request->getHeaderLine('Content-Type'));
+
+        if ($type === 'application/x-www-form-urlencoded') {
```

Comment:
Feel free to copy from here :) https://github.com/slimphp/Slim/blob/3.x/Slim/Http/Request.php#L204-L241

Great work btw :)

Reply:
See #225 :-)

Reply:
https://github.com/WyriHaximus/reactphp-http-middleware-custom-request-body-parsers :sunglasses: